node/db/src/rocks.rs

```diff
@@ -91,7 +92,12 @@ impl Store for RocksDb {
     where
         K: AsRef<[u8]>,
     {
-        self.db.get(key).map_err(Error::from)
+        let key = key.as_ref();
+        if !self.db.key_may_exist(key) {
```

Comment:
Maybe consider adding some comment here, it may not be obvious, ie:

`// If the return value of 'key_may_exist' is false, the key definitely does not exist in the db`

Reply:
Fixed